src/scss/_objects.content-card.scss

```diff
@@ -0,0 +1,41 @@
+.content-card {
+  margin: halve($inuit-base-spacing-unit) 0;
```

Comment:
I believe a more "Grommet" way of doing things would be to add a wrapper `Box` with the appropriate `pad` values (such as `pad={{ vertical: 'small', horizontal: 'none' }}` in this case), instead of adding custom styles for the margin.  Would that work here?